packages/shared/components/atoms/BalanceSummaryRow.svelte

```diff
@@ -0,0 +1,69 @@
+<script lang="ts">
+    import { FontWeight, Text } from 'shared/components'
+
+    export let title: string
+    export let subtitle: string = ''
+    export let bold: boolean = false
+    export let amount: string
+    export let convertedAmount: string
```

Comment:
Why don't you need default values for all of them?

Reply:
because the other ones are (for now) required. This ui element currently doesnt exist without e.g a title, amount or converted amount. But it exists with an empty subtitle